clang/test/SemaHLSL/noinline.hlsl

```diff
@@ -0,0 +1,13 @@
+// RUN: %clang_cc1 -triple dxil-pc-shadermodel6.0-compute -emit-llvm-only %s -verify
```

Comment:
We should not need to emit IR to test a warning. This should work with `-fsyntax-only` or without the `-emit-llvm-only` flag.